packages/client/cypress/e2e/detail-view.spec.ts

```diff
@@ -2,16 +2,39 @@
 
 describe("detail view page tests", () => {
 	beforeEach("match query and stub response", () => {
-		cy.intercept(
-			"GET",
-			"https://losh.ose-germany.de/w/api.php?action=wbgetentities&ids=Q513&format=json&origin=*",
-			{ fixture: "getEntityResponse" }
-		).as("getEntity");
-		cy.visit("/detail/Q513");
-		cy.wait("@getEntity");
+		cy.fixture("detail-view-fixtures").then((data) => {
+			const { getDetailsRes, getAllPost } = data;
+			cy.intercept("POST", "http://localhost:3000/graphql", (req) => {
+				req.reply((res) => {
```

Comment:
There is documentation in Cypress about how to specifically intercept GraphQL requests :) 

https://docs.cypress.io/guides/testing-strategies/working-with-graphql#Expectations-for-Query-or-Mutation-Results